server/views/pages/temporaryabsences/confirmTemporaryAbsenceAddedToRoll.njk

```diff
@@ -1,4 +1,5 @@
 {% extends "../../partials/layout.njk" %}
+{% set backNavigationLink = { hasBackLink: false } %}
```

Comment:
very meh.. but worth pointing out, here you don't have the same thing with primitives - the variable name can provide the context you need, so you could get away with:
```
{% set hasBackLink = false %}
```

as where you use it can refer to that name:

```
{% if hasBackLink !== false %}
```

Reply:
thanks - sorted in latest commit